### ISSUE TITLE
During training, the number of displayed training data is reduced by half

### ISSUE THREAD

**Issue by cena001plus** [open-mmlab/mmdetection#6437]
HI, my training dataset quantity is 9712，
when use the model of atss_r101 , i found   the number of training dataset is not right,  it change to 4856    ,    it's right when train faster_rcnn_r50_fpn_1x.

`0.1116, loss_centerness: 0.5863, loss: 0.7148
2021-11-04 11:35:13,545 - mmdet - INFO - Epoch [12][100/4856]	lr: 1.000e-04, eta: 0:09:59, time: 0.214, data_time: 0.004, memory: 2523, loss_cls: 0.0174, loss_bbox: 0.1116, loss_centerness: 0.5859, loss: 0.7149
2021-11-04 11:35:24,157 - mmdet - INFO - Epoch [12][150/4856]	lr: 1.000e-04, eta: 0:09:48, time: 0.212, data_time: 0.004, memory: 2523, loss_cls: 0.0175, loss_bbox: 0.1110, loss_centerness: 0.5846, loss: 0.7131
2021-11-04 11:35:34,852 - mmdet - INFO - Epoch [12][200/4856]	lr: 1.000e-04, eta: 0:09:38, time: 0.214, data_time: 0.004, memory: 2523, loss_cls: 0.0170, loss_bbox: 0.1071, loss_centerness: 0.5851, loss: 0.7092
2021-11-04 11:35:45,481 - mmdet - INFO - Epoch [12][250/4856]	lr: 1.000e-04, eta: 0:09:27, time: 0.213, data_time: 0.004, memory: 2523, loss_cls: 0.0176, loss_bbox: 0.1065, loss_centerness: 0.5851, loss: 0.7092
`

**Comment by RangiLyu** [comment]
4856 is the number of total iterations of one epoch. It seems that you are using batchsize=2, so it is normal.

**Comment by cena001plus** [comment]
ok

**Comment by cena001plus** [comment]
ok
